example/docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+---
+version: "3"
+
+services:
+  azurite:
+    image: mcr.microsoft.com/azure-storage/azurite:3.29.0
+    container_name: "azurite"
+    hostname: azurite
+    command: "azurite-blob --blobHost 0.0.0.0 --blobPort 10000"
+    ports:
+      - "10000:10000"
+    healthcheck:
+      test: nc 127.0.0.1 10000 -z
+      interval: 1s
+      retries: 30
+
+  azurite-seed:
+    image: rclone/rclone:1.65
+    depends_on:
+      azurite:
+        condition: service_healthy
+    volumes:
+      - ./:/example
+    environment:
+      - RCLONE_CONFIG_BLOBS_TYPE=azureblob
+      - RCLONE_CONFIG_BLOBS_ENDPOINT=http://azurite:10000/devstoreaccount1
+      - RCLONE_CONFIG_BLOBS_USE_EMULATOR=true
+    entrypoint:
+      - sh
+      - -c
+      - |
+        rclone copy -vv /example/blob-inventory blobs:blob-inventory
+
+  azure-storage-usage-exporter:
+    image: azure-storage-usage-exporter:local  # run local image if available (used in CI)
```

Comment:
Mooi, je zou nu dit example in de CI kunnen starten, als een e2e test. Dan weten we zeker dat het blijft werken